### PR TITLE
Add windows dark mode toggle

### PR DESCRIPTION
**Pull request by Garulf**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -429,5 +429,17 @@
         "UrlSourceCode": "https://github.com/Garulf/window-services/tree/main",
         "IcoPath": "https://github.com/Garulf/window-services/raw/c78cdb4b51c62fb09d860dad143c8773c686c967/icons/cog.png",
         "ETag": "W/\"b5cdd2d4b168063f5f772efb45ba5b3c15f8ed705b14cf4e9e212b106f1a261d\""
+    },
+    {
+        "ID": "4F31D5DF26984581BDF4213D40860ADF",
+        "Name": "Windows Dark Mode Toggle",
+        "Description": "Search and cast your Plex Media Library",
```
Comment (jjw24) [inline plugins.json:436]: the description is the same as Plexy?

File: plugins.json [hunk 2]
```diff
@@ -429,5 +429,17 @@
...
+        "Author": "Garulf",
+        "Version": "1.0.0",
+        "Language": "python",
+        "Website": "https://github.com/Garulf/windows-dark-mode-toggle",
+        "UrlDownload": "https://github.com/Garulf/windows-dark-mode-toggle/releases/download/v1.0.0/windows-dark-mode-toggle.zip",
+        "UrlSourceCode": "https://github.com/Garulf/windows-dark-mode-toggle/tree/main",
+        "IcoPath": "./icon/light-dark.png"
```
Comment (jjw24) [inline plugins.json:443]: this needs to be url

**Review by jjw24 — CHANGES_REQUESTED** [review]
minor fixes required :)

**Comment by jjw24** [comment]
also please resolve conflict

File: plugins.json (hunk 1 above)
Comment (Garulf) [reply plugins.json]: Fixed

**Comment by Garulf** [comment]
Thanks for catching that! I guess rushing it out before bed was a bad idea.

**Review by jjw24 — DISMISSED** [review]

**Review by jjw24 — APPROVED** [review]

**Comment by Garulf** [comment]
Thanks for resolving the conflict. I had to go AFK.

**Pull request merged**